### PR TITLE
Update CoC Requirements

### LINKED ISSUE #786: Clarify requirements around CoC violation
Currently there are no requirements beyond adopting the Foundation's CoC for projects.

It seems we would need additional requirements for projects who wish to handle CoC violations themselves. For example, having a clear process for raising a violation, certain guarantees around privacy, etc.

---

Next steps:

- [x] open a pull request against [FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS](https://github.com/openjs-foundation/cross-project-council/blob/main/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md) that adds a section listing the requirements outlined above (see #1135).
- [ ] open a pull request against the [onboarding checklist](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/PROJECT_PROGRESSION.md#onboarding-checklist) that includes implementing those requirements.
  - [ ] have an email reporting address
  - [ ] (required for impact and at-large) have at least more than one person selected by the project maintainership (through a process defined by the project) on that email address
  - [ ] publicly list who is on that email address
  - [ ] publicly document the decision-making process
  - [ ] confidentiality of reporter and victim
  - [ ] basic rules around data retention to meet legal requirements (e.g. GDPR) 
- [ ] following up on the audit @rginn is running to make sure that the projects that have opted in to run their own CoC enforcement have implemented those requirements.

### PR DESCRIPTION
**Pull request by tobie**
Closes #786.

### CODE REVIEW

**Review by mikesamuel — APPROVED** [review]
Looks good.  Terminology fixes are done consistently.  Abbreviations like *CoCP* are defined before first use and/or are widely used.

**Review by mhdawson — APPROVED** [review]
LGTM

File: FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
 # Foundation Code of Conduct Requirements
 
 It is important that there be a Code of Conduct (CoC) which
-is documented and enforced for the OpenJS Foundation, 
-hereinafter referred to as the "Foundation", and the
+is documented and enforced for the OpenJS Foundation and the
 projects within it. This document defines the requirements
-for a CoC for the Foundation and the projects which are part
+for a CoC for the foundation and the projects which are part
 of the foundation.
```
Comment (Trott) [inline FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md:7]: Optional suggestion to focus it a bit more:

```suggestion
This document defines the requirements for a Code of Conduct (CoC) for
projects within the OpenJS Foundation.
```

**Review by Trott — APPROVED** [review]
LGTM with or without my suggestion.

File: FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md (hunk 1 above)
Comment (tobie) [reply FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md:7]: Can't seem to commit your changes here, @Trott, but I'll include them manually.

**Comment by tobie** [comment]
Closed in favor of https://github.com/openjs-foundation/cross-project-council/pull/984/commits/ab239973f964685403bd02a2d0e7544ed598daaa.

**Pull request closed without merging**